ui/src/components/VaultSummary.jsx

```diff
@@ -94,13 +93,15 @@ export function VaultSummary({ vault, brandToInfo, id }) {
   } = makeDisplayFunctions(brandToInfo);
 
   const {
-    debt, // amount
+    debtSnapshot, // amount
     interestRate, // ratio
     liquidationRatio, // ratio
     locked, // amount
     status, // string
```

Comment:
Related to your above comment, vault is type `VaultData` [ui/src/types/types.js](https://github.com/Agoric/dapp-treasury/pull/94/files/1fdcce49c6f8b9d4d62327f9ab2cdd7bbdcc272a#diff-eb2331b2e81f8a38629f121d4974ca0d4c7844e4c61a481d853f980f3c7b9a42) which is slightly different from `VaultUIState`. It would be elegant to have these be the same types, but I'm unsure if we should try to make them conform.

For example, the `status` is partially informed by the `VaultPhase`, but it can also be something like `Error in Offer`, `Loading`, or `Pending wallet approval`. These mean different things to the UI, but aren't part of the contract. There's also the `err` property.

In Web2 applications I've worked on, it's been standard practice for the UI to have its own types, since inevitably there's always transformations that need to be done to backend data to make it convenient/usable to the UI code. I guess with OCAPS this line is a bit blurred, and the transformations are minimal in some places, but I think it will be easier to not paint ourselves into a corner if we maintain this separation.

Reply:
Changed as discussed to use more of the types from run-protocol where appropriate